beastling/extractor.py

```diff
@@ -1,7 +1,6 @@
 import xml.etree.ElementTree as ET
 from pathlib import Path
-
-from clldutils.inifile import INI
+from configparser import ConfigParser
```

Comment:
Oh, the PR also removes the dependency on `clldutils`.

Reply:
I don't know if GitHub's PR review interface is correctly showing me the final result of all your commits, but I see a ` from clldutils.inifile import INI` at line 107 of `beastxml.py`, so are we not quite there yet?

Reply:
Ha, true! The import isn't used anywhere, so we are there, just have to remove the import - and probably add some tests ...